.github/workflows/node-flow-deploy-release-artifact.yaml

```diff
@@ -42,72 +44,93 @@ defaults:
 
-  release-tag:
-    name: Release [Tag]
-    uses: ./.github/workflows/node-zxc-build-release-artifact.yaml
-    needs:
-      - prepare-tag-release
-    with:
-      version-policy: specified
-      new-version: ${{ needs.prepare-tag-release.outputs.version }}
-      trigger-env-deploy: none
-      release-profile: ${{ needs.prepare-tag-release.outputs.prerelease == 'true' && 'PrereleaseChannel' || 'MavenCentral' }}
-    secrets:
-      access-token: ${{ secrets.GITHUB_TOKEN }}
-      bucket-name: ${{ secrets.RELEASE_ARTIFACT_BUCKET_NAME }}
-      cdn-bucket-name: ${{ secrets.CDN_ARTIFACT_BUCKET_NAME }}
-      svcs-ossrh-username: ${{ secrets.SVCS_OSSRH_USERNAME }}
-      svcs-ossrh-password: ${{ secrets.SVCS_OSSRH_PASSWORD }}
-      svcs-gpg-key-contents: ${{ secrets.SVCS_GPG_KEY_CONTENTS }}
-      svcs-gpg-key-passphrase: ${{ secrets.SVCS_GPG_KEY_PASSPHRASE }}
-      sdk-ossrh-username: ${{ secrets.PLATFORM_OSSRH_USERNAME }}
-      sdk-ossrh-password: ${{ secrets.PLATFORM_OSSRH_PASSWORD }}
-      sdk-gpg-key-contents: ${{ secrets.PLATFORM_GPG_KEY_CONTENTS }}
-      sdk-gpg-key-passphrase: ${{ secrets.PLATFORM_GPG_KEY_PASSPHRASE }}
-      slack-webhook-url: ${{ secrets.PLATFORM_SLACK_RELEASE_WEBHOOK }}
-      jf-url: ${{ vars.JF_URL }}
-      jf-docker-registry: ${{ vars.JF_DOCKER_REGISTRY }}
-      jf-user-name: ${{ vars.JF_USER_NAME }}
-      jf-access-token: ${{ secrets.JF_ACCESS_TOKEN }}
+      - name: Call Build Release Artifact
+        if: ${{ steps.semantic.outputs.new_release_published == 'true' }}
+        uses: ./.github/workflows/node-zxc-build-release-artifact.yaml
+        with:
+          version-policy: specified
+          trigger-env-deploy: none
+          release-profile: ${{ contains(steps.semantic.outputs.new_release_version, '-') && 'PrereleaseChannel' || 'MavenCentral' }}
+          new-version: ${{ steps.semantic.outputs.new_release_version }}
+        secrets:
+          access-token: ${{ secrets.GITHUB_TOKEN }}
+          bucket-name: ${{ secrets.RELEASE_ARTIFACT_BUCKET_NAME }}
+          cdn-bucket-name: ${{ secrets.CDN_ARTIFACT_BUCKET_NAME }}
+          svcs-ossrh-username: ${{ secrets.SVCS_OSSRH_USERNAME }}
+          svcs-ossrh-password: ${{ secrets.SVCS_OSSRH_PASSWORD }}
+          svcs-gpg-key-contents: ${{ secrets.SVCS_GPG_KEY_CONTENTS }}
+          svcs-gpg-key-passphrase: ${{ secrets.SVCS_GPG_KEY_PASSPHRASE }}
+          sdk-ossrh-username: ${{ secrets.PLATFORM_OSSRH_USERNAME }}
+          sdk-ossrh-password: ${{ secrets.PLATFORM_OSSRH_PASSWORD }}
+          sdk-gpg-key-contents: ${{ secrets.PLATFORM_GPG_KEY_CONTENTS }}
+          sdk-gpg-key-passphrase: ${{ secrets.PLATFORM_GPG_KEY_PASSPHRASE }}
+          slack-webhook-url: ${{ secrets.PLATFORM_SLACK_RELEASE_WEBHOOK }}
+          jf-url: ${{ vars.JF_URL }}
+          jf-docker-registry: ${{ vars.JF_DOCKER_REGISTRY }}
+          jf-user-name: ${{ vars.JF_USER_NAME }}
+          jf-access-token: ${{ secrets.JF_ACCESS_TOKEN }}
```

Comment:
I don't think I understand why we're going through a major refactor on this file. Can we re-implement this again so that the Prepare Release [Tag] job exists and utilizes semantic release in a way similar to [hashgraph/full-stack-testing repository](https://github.com/hashgraph/full-stack-testing/blob/main/.github/workflows/flow-deploy-release-artifact.yaml)? I don't think we need to rip the release-tag steps out.